pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
```

Comment:
I think `front-end web development` is accurate. From my experience working with web dev, `front-end` is user facing, but the user might be a `back-end` service.

Sure, it's common for recruiters to refer to `front end devs` to people who can juggle some css/js, but being engineers, the semantics of front and back end mean nothing but what is exposed.

Interestingly, by that definition, `front end dev` and `API dev` would be fairly similar :stuck_out_tongue: since an API is the front end of some service.